webserver/src/main/java/io/kestra/webserver/services/ai/MetricChatModelListener.java

```diff
@@ -0,0 +1,87 @@
+package io.kestra.webserver.services.ai;
+
+import dev.langchain4j.model.chat.listener.ChatModelErrorContext;
+import dev.langchain4j.model.chat.listener.ChatModelListener;
+import dev.langchain4j.model.chat.listener.ChatModelResponseContext;
+import dev.langchain4j.model.chat.request.ChatRequest;
+import dev.langchain4j.model.chat.response.ChatResponse;
+import dev.langchain4j.model.output.TokenUsage;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.MeterRegistry;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+@Singleton
+@Slf4j
+public class MetricChatModelListener implements ChatModelListener {
+    @Inject
+    private MeterRegistry meterRegistry;
+
+    @Override
+    public void onResponse(ChatModelResponseContext responseContext) {
+        ChatRequest request = responseContext.chatRequest();
+        ChatResponse response = responseContext.chatResponse();
+
+        String[] tags = new String[]{
+            "messages_count", String.valueOf(request.messages().size()),
```

Comment:
```suggestion
```

A count should never be a tag as it may create a very big cardinality of metrics